src/Microsoft.ComponentDetection.Orchestrator/Services/DetectorProcessingService.cs

```diff
@@ -23,6 +23,8 @@ namespace Microsoft.ComponentDetection.Orchestrator.Services;
 
 public class DetectorProcessingService : IDetectorProcessingService
 {
+    private const int DefaultMaxDetectionThreads = 10;
```

Comment:
We should probably be a bit more conservative with the default here - maybe 3-5

Reply:
Since some of the detector processes are reaching out to APIs for artifact information, we don't want to have them get throttled for too many concurrent requests.

Reply:
makes sense, updated